commons/connector/oura/oura_heart_rate.avsc

```diff
@@ -0,0 +1,14 @@
+{
+  "namespace": "org.radarcns.connector.oura",
+  "name": "OuraHeartRate",
+  "type": "record",
+  "doc": "The Heart Rate data scope includes time-series heart rate data throughout the day and night. Heart rate is provided at 5-minute increments.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Start time of the activity in seconds since January 1, 1970, 00:00:00 UTC (Unix timestamp)."},
+    {"name": "timeReceived", "type": "double", "doc": "Time that this record was collected by a service in seconds since the Unix Epoch (s)."},
+    {"name": "bpm", "type": ["null", "int"], "doc": "Heart rate in beats per minute.", "default": null},    
+    {"name": "source", "type":
+      {"name": "OuraHeartRateSource", "type": "enum", "symbols": ["AWAKE", "REST", "SLEEP", "SESSION", "LIVE", "UNKNOWN"], "doc": "Heart rate source type as computed by Oura."},
+      "doc": "Heart rate source type as computed by Oura.", "default": "UNKNOWN" }
```

Comment:
Add docs what `UNKNOWN` means (e.g. when source isn't provided when heart rate data is from sleep api, etc)